giraffe/src/components/SimpleTable/PagedTable.tsx

```diff
@@ -231,8 +233,30 @@ const PagedTable: FC<Props> = ({result, properties}) => {
     setTotalPages,
   } = useContext(PaginationContext)
   const [height, setHeight] = useState(0)
+  const [headerHeight, setHeaderHeight] = useState(0)
+  const [rowHeight, setRowHeight] = useState(0)
   const ref = useRef()
 
+  if (headerHeight === 0) {
+    const calculatedHeaderHeight =
+      document.querySelector<HTMLElement>(`#${pagedTableHeaderId}`)
```

Comment:
I already mentioned using the querySelector vs a ref in the other PR, so I won't repeat it.